common/src/main/java/com/wynntils/core/events/EventThread.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.events;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface EventThread {
+    Type value() default Type.RENDER;
+
+    enum Type {
+        RENDER,
+        IO,
+        WORKER,
+        ANY
```

Comment:
Please comment what these are. I think Render and IO are minecraft internal, and Worker is artemis internal, but once again it's hard to know.

Reply:
Ok, fixed.